lincheck/src/main/java/org/jetbrains/kotlinx/lincheck/strategy/Strategy.java

```diff
@@ -46,18 +52,22 @@
 public abstract class Strategy {
     protected final ExecutionScenario scenario;
     protected final Reporter reporter;
-    private final Verifier verifier;
+    protected final Verifier verifier;
 
     protected Strategy(ExecutionScenario scenario, Verifier verifier, Reporter reporter) {
         this.scenario = scenario;
         this.verifier = verifier;
         this.reporter = reporter;
     }
 
-    protected void verifyResults(ExecutionResult results) {
+    protected void verifyResults(ExecutionResult results, List<ThreadEvent> threadEvents) {
         if (!verifier.verifyResults(results)) {
             StringBuilder msgBuilder = new StringBuilder("Invalid interleaving found:\n");
             appendIncorrectResults(msgBuilder, scenario, results);
+            if (threadEvents != null) {
+                msgBuilder.append(System.lineSeparator());
```

Comment:
you use `\n` two lines above :)
Either use `System.lineSeparator()` or `\n` everywhere